### PR TITLE
Get output admin units from db

### PR DESCRIPTION
**Pull request by javierps**

### CODE REVIEW

File: packages/taxdat/R/covariate_helpers.R [hunk 1]
```diff
@@ -1,2002 +1,2057 @@
...
+#' Pulls the admin units based on the iso3 code using rgeoboundaries
+#' @param iso_code 
+#' @param admin_level 
+#'
+#' @return an sf object
+#' @export
+#'
+get_country_admin_units <- function(iso_code, 
+                                    admin_level = 1) {
+  library(sf)
```
Comment (eclee25) [inline packages/taxdat/R/covariate_helpers.R:1361]: why are we library-ing here?

File: packages/taxdat/R/covariate_helpers.R [hunk 2]
```diff
@@ -1,2002 +1,2057 @@
...
+                                             dbuser = dbuser))
+    if (!inherits(adm_sf, "try-error")) {
+      return(adm_sf)
+    }
+  }
+  
+  if (clip_to_adm0 & !(0 %in% admin_levels)) {
+    stop("Admin level 0 needs to be included if clip_to_adm0 = TRUE.")
+  }
+
```
Comment (eclee25) [inline packages/taxdat/R/covariate_helpers.R:1478]: Even though we used get_country_admin_units to generate the shapes that were loaded to get_country_admin_units_db, I'm concerned that we could edit the complex conditions under get_country_admin_units and then forget that we are loading the already-pulled shapes from the covariate database. We should at least add some sort of warning that the shapes are being re-pulled from external APIs if this part is triggered.

**Review by eclee25 — CHANGES_REQUESTED** [review]
Could you also push the script that was used to generate and upload the output shapefiles to the cholera-covariates repo (Layers/)? I'd like to look at that at the same time

File: packages/taxdat/R/covariate_helpers.R (hunk 1 above)
Comment (javierps) [reply packages/taxdat/R/covariate_helpers.R:1361]: Removed this.

File: packages/taxdat/R/covariate_helpers.R (hunk 2 above)
Comment (javierps) [reply packages/taxdat/R/covariate_helpers.R:1478]: Good point. I will add a hash of the code of get_country_admin_units to check whether the scripts are consistent.

**Pull request merged**